### ISSUE TITLE
Adding .env support for Configs

### ISSUE THREAD

**Issue by arifulhb** [syaifulsz/sesimple-framework#1]
Great initiative and can be a micro framework with some detailed documentation and database support. Current configs are hard coded and all data are available in public.

I've forked the framework and in few days will add `.env` support for config files. Hopefully, it will help.

**Comment by syaifulsz** [comment]
Yeah, sure thanks @arifulhb ! For other project, I'm actually extended the framework quite bit by other vendor via composer depends on requirement. For database, I've migrated illuminate/database#5.6 independently. Works like charm!

P/S: https://www.tengokwayang.com is actually using the same framework.

**Comment by arifulhb** [comment]
Building an application without `.env` is fine, but it's necessary when you want to opensource your project. It will save from hassle of removing credentials from the app before pushing to GitHub or other git repo.
